---
File: README.md
Lecture: Advanced Bioinformatics 1
Professor: Martin Steinegger (martin.steinegger@snu.ac.kr)
TA: Sukhwan Park (pskvins@snu.ac.kr)
Last Modified: 2023-09-06 20:59:05
---

# Advanced Bioinformatics 1
# The paper pool for Advanced Bioinformatics 2021

## Annotation
1. Augustus [paper](https://academic.oup.com/bioinformatics/article/27/6/757/234821), [github](https://github.com/Gaius-Augustus/Augustus)
2. BRAKER2 [paper](https://academic.oup.com/nargab/article/3/1/lqaa108/6066535), [github](https://github.com/Gaius-Augustus/BRAKER)
3. Prodigal [paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-11-119), [github](https://github.com/hyattpd/Prodigal)

## Genome comparison
1. mummer1-4 [paper](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005944), [github](https://github.com/mummer4/mummer)
2. mash [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0997-x), [github](https://github.com/marbl/mash)
3. FastANI [paper](https://www.nature.com/articles/s41467-018-07641-9), [github](https://github.com/ParBLiSS/FastANI)

## Multiple sequence alignment (MSA)
1. clustalomega [paper](https://www.embopress.org/doi/full/10.1038/msb.2011.75), [github](https://github.com/GSLBiotech/clustal-omega), [homepage](http://www.clustal.org/omega/)
2. Muscle5 [paper](https://www.nature.com/articles/s41467-022-34630-w), [github](https://github.com/rcedgar/muscle), [homepage](https://www.drive5.com/muscle/)
3. Progressive Cactus [paper1](https://genome.cshlp.org/content/21/9/1512.short), [paper2](https://www.nature.com/articles/s41586-020-2871-y), [github](https://github.com/ComparativeGenomicsToolkit/cactus)

## Homology search
1. mmseqs2 [paper](https://www.nature.com/articles/nbt.3988), [github](https://github.com/soedinglab/MMseqs2)
2. Diamond1+2 [paper1](https://www.nature.com/articles/s41592-021-01101-x), [paper2](https://www.nature.com/articles/nmeth.3176), [github](https://github.com/bbuchfink/diamond)
3. HHblits [paper1](https://www.nature.com/articles/nmeth.1818), [paper2](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3019-7), [github](https://github.com/soedinglab/hh-suite)
4. Foldseek [paper](https://www.nature.com/articles/s41587-023-01773-0), [github](https://github.com/steineggerlab/foldseek)
5. tm-align [paper](https://academic.oup.com/nar/article/33/7/2302/2401364), [website](https://zhanggroup.org/TM-align/)

## Transcript
1. kallisto [paper](https://www.nature.com/articles/nbt.3519), [github](https://github.com/pachterlab/kallisto), [homepage](https://pachterlab.github.io/kallisto/)
2. salmon [paper](https://www.nature.com/articles/nmeth.4197), [github](https://github.com/COMBINE-lab/salmon), [homepage](https://combine-lab.github.io/salmon/)
3. Oases [paper](https://academic.oup.com/bioinformatics/article/28/8/1086/195757), [github](https://github.com/dzerbino/oases)
4. CellChat [paper](https://www.nature.com/articles/s41467-021-21246-9#Sec12), [github](https://github.com/sqjin/CellChat)

## Metagenomics
1. kraken-uniq [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1568-0), [github](https://github.com/fbreitwieser/krakenuniq)
2. HUMAnN2 [paper](https://www.nature.com/articles/s41592-018-0176-y), [github](https://github.com/biobakery/humann), [homepage](https://huttenhower.sph.harvard.edu/humann2/)
3. Kaiju [paper](https://www.nature.com/articles/ncomms11257#Sec9), [github](https://github.com/bioinformatics-centre/kaiju), [homepage](https://bioinformatics-centre.github.io/kaiju/)

## Phylogenetics
1. raxml [paper](https://academic.oup.com/bioinformatics/article/30/9/1312/238053), [github](https://github.com/stamatak/standard-RAxML), [homepage](https://cme.h-its.org/exelixis//web/software/raxml/index.html)
2. iqtree [paper](https://academic.oup.com/mbe/article/37/5/1530/5721363), [github](https://github.com/Cibiv/IQ-TREE), [homepage](http://www.iqtree.org/)
3. MrBayes [paper](https://academic.oup.com/bioinformatics/article/17/8/754/235132), [homepage](https://nbisweden.github.io/MrBayes/index.html)

## RNA structure prediction
1. viennarna [paper](https://almob.biomedcentral.com/articles/10.1186/1748-7188-6-26), [github](https://github.com/ViennaRNA/ViennaRNA), [homepage](https://www.tbi.univie.ac.at/RNA/)
2. R-scape [paper](https://www.nature.com/articles/nmeth.4066), [software](http://www.eddylab.org/software.html), [homepage](http://www.eddylab.org/R-scape/)
3. UFold [paper](https://academic.oup.com/nar/article/50/3/e14/6430845), [github](https://github.com/uci-cbcl/UFold)

## Protein structure
1. RoseTTAFold [paper](https://www.science.org/doi/abs/10.1126/science.abj8754), [github](https://github.com/RosettaCommons/RoseTTAFold)
2. gromacs [paper](https://www.sciencedirect.com/science/article/pii/S2352711015000059?via%3Dihub), [github](https://github.com/gromacs/gromacs), [homepage](https://www.gromacs.org/)
3. ProteinMPNN [paper](https://www.science.org/doi/10.1126/science.add2187), [github](https://github.com/dauparas/ProteinMPNN)
4. ESMFold [paper](https://www.science.org/doi/10.1126/science.ade2574), [github](https://github.com/facebookresearch/esm), [homepage](https://esmatlas.com/)

## Workflow and package
1. nextflow [paper](https://www.nature.com/articles/nbt.3820), [github](https://github.com/nextflow-io/nextflow), [homepage](https://www.nextflow.io/)
2. snakemake [paper](https://academic.oup.com/bioinformatics/article/28/19/2520/290322), [github](https://github.com/snakemake/snakemake), [documentation](https://snakemake.readthedocs.io/en/stable/)
3. bioconda [paper](https://www.nature.com/articles/s41592-018-0046-7), [github](https://github.com/bioconda)

## Assembler
1. hifiasm [paper](https://www.nature.com/articles/s41592-020-01056-5), [github](https://github.com/chhylp123/hifiasm)
2. Verkko [paper](https://www.nature.com/articles/s41587-023-01662-6), [github](https://github.com/marbl/verkko/)

## Read aligner
1. hisat1+2 [paper](https://www.nature.com/articles/s41587-019-0201-4), [github](https://github.com/DaehwanKimLab/hisat2), [homepage](http://daehwankimlab.github.io/hisat2/)
2. minimap2 [paper](https://academic.oup.com/bioinformatics/article/34/18/3094/4994778), [github](https://github.com/lh3/minimap2)
3. vsearch [paper](https://peerj.com/articles/2584/), [github](https://github.com/torognes/vsearch)

## Language models
1. ProtTrans [paper](https://ieeexplore.ieee.org/document/9477085), [github](https://github.com/agemagician/ProtTrans)
2. ESM [paper](https://www.biorxiv.org/content/10.1101/622803v4), [github](https://github.com/facebookresearch/esm)
3. MSA Transformer [paper](https://proceedings.mlr.press/v139/rao21a.html), [github](https://github.com/facebookresearch/esm)

## Variant caller
1. freebayes [paper](https://arxiv.org/abs/1207.3907), [github](https://github.com/freebayes/freebayes)
2. pilon [paper](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0112963), [github](https://github.com/broadinstitute/pilon)
3. Manta [paper](https://academic.oup.com/bioinformatics/article/32/8/1220/1743909?login=true), [github](https://github.com/Illumina/manta)

## Pangenomes
1. VG [paper](https://www.nature.com/articles/nbt.4227), [github](https://github.com/vgteam/vg)
2. Minigraph [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02168-z), [github](https://github.com/lh3/minigraph)

## Other topics
1. Conterminator [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02023-1), [github](https://github.com/martin-steinegger/conterminator)
2. GTDB [paper1](https://academic.oup.com/nar/article/50/D1/D785/6370255), [paper2](https://academic.oup.com/bioinformatics/article/36/6/1925/5626182), [github](https://github.com/ecogenomics/gtdbtk)
